crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs

```diff
@@ -3542,11 +3542,16 @@ impl Analyzer<'_, '_> {
                     }
                     .into())
                 } else {
-                    Err(ErrorKind::NoSuchVar {
-                        span,
-                        name: i.clone().into(),
+                    match &*i.sym {
+                        "any" | "never" | "unknown" | "string" | "number" | "bigint" | "boolean" | "undefined" | "symbol" => {
+                            Err(ErrorKind::CannotExportNonLocalVar { span }.into())
```

Comment:
We should not do this here

Reply:
We should do this check from https://github.com/dudykr/stc/blob/bc089c22e747a4798f9ea80fba9ac5a011b10677/crates/stc_ts_file_analyzer/src/analyzer/export.rs#L322-L329 instead